tests/conftest.py

```diff
@@ -64,11 +64,12 @@ def ocp_target_version(request):
 
 
 @pytest.fixture(scope="session")
-def rosa_regions(rosa_allowed_commands):
+def rosa_regions(rosa_allowed_commands, ocm_token):
     # A region (any region) is required for ROSA commands
     return rosa.cli.execute(
         command="list regions --region us-west-2",
```

Comment:
`--region us-west-2` this will fail with the new changes.
remove it from the command